### PR TITLE
Fixed broken links

### PR DESCRIPTION
**Pull request by catherinepope**
# Description
Fixed broken links.

# Reasons
Two links were pointing to non-existent PDFs.

### CODE REVIEW

File: jekyll/_cci2/security-server.adoc [hunk 1]
```diff
@@ -98,7 +98,7 @@ If you are getting started with CircleCI, there are some points you can ask your
...
-* If your project is open source and public, please make note of whether you want to share your environment variables. On CircleCI, you can change a project's settings to control whether your environment variables can pass on to _forked versions of your repo_. This is **not enabled** by default. You can read more about these settings and open source security in our <<oss#security,Open Source Projects document>>.
\ No newline at end of file
+* If your project is open source and public, please make note of whether you want to share your environment variables. On CircleCI, you can change a project's settings to control whether your environment variables can pass on to _forked versions of your repo_. This is **not enabled** by default. You can read more about these settings and open source security in our https://circleci.com/docs/2.0/oss/index.html#security[Open Source Projects Document].
```
Comment (rosieyohannan) [inline jekyll/_cci2/security-server.adoc:104]: GH won't let me add a suggested change for some reason, but we don't need the index.html in there, then good to go. Thanks so much for finding these issues :-D

Comment (rosieyohannan) [inline jekyll/_cci2/security-server.adoc:104]:
```suggestion
* If your project is open source and public, please make note of whether you want to share your environment variables. On CircleCI, you can change a project's settings to control whether your environment variables can pass on to _forked versions of your repo_. This is **not enabled** by default. You can read more about these settings and open source security in our https://circleci.com/docs/2.0/oss/i#security[Open Source Projects Document].
```

**Comment by rosieyohannan** [comment]
Just one thing then good to go. Thanks for finding these issues!

**Comment by devinmarieb** [comment]
@rosieyohannan @catherinepope Can someone tell me why this needs to be done? These links are the proper format for internal linking and do in fact work. I just double checked on the docs that are live now, and they are working fine.

**Comment by catherinepope** [comment]
The links weren't working in the generated PDFs, Devin.

**Comment by devinmarieb** [comment]
@catherinepope Will the trade off here be that the docs will now show external links for these sources with the external linking icon?

**Comment by catherinepope** [comment]
I guess so. Most of the other links were already in that format (i.e. showing as external). Is this right, @rosieyohannan ?

**Comment by rosieyohannan** [comment]
No they don't show as external links thankfully. Unfortunately if we use relative links at the moment it's breaking the PDFs. I need to investigate.

**Comment by devinmarieb** [comment]
@rosieyohannan @catherinepope Thanks! Good to know. I try not to touch the server docs, but I did have to update an out of date link once and the format that was already there was the internal linking format. I'll see if I can find that PR and double check that link is fine.

**Comment by rosieyohannan** [comment]
I thought it was fixed by Nav a while back but I realised while working on this release that the PDF links were just printing the liquid tags rather than rendering as links. I'm sure there is a workaround but needed to get them fixed for the release last night 😅

**Comment by rosieyohannan** [comment]
If there's anything weird in server docs it's almost always got something to do with PDFs unfortunately. Thanks for spotting this though @devinmarieb, defo needs fixing.

**Comment by rosieyohannan** [comment]
The asciidoc link format only works if the link is to a page in the same PDF, for example, otherwise the link would have to actually open a file. And as we have separate installation and ops guides this happens quite a bit.

File: jekyll/_cci2/security-server.adoc [hunk 2]
```diff
@@ -98,7 +98,7 @@ If you are getting started with CircleCI, there are some points you can ask your
   ** literally printing secrets in your codebase or in your shell with `echo`.
   ** programs or debugging tools that print secrets on error.
 * Consult your VCS provider's permissions for your organization (if you are in an organization) and try to follow the https://en.wikipedia.org/wiki/Principle_of_least_privilege[Principle of Least Privilege].
-* Use Restricted Contexts with teams to share environment variables with a select security group. Read through the <<contexts#restricting-a-context,contexts>> document to learn more.
+* Use Restricted Contexts with teams to share environment variables with a select security group. Read through the https://circleci.com/docs/2.0/contexts#restricting-a-context[contexts] document to learn more.
```
Comment (rosieyohannan) [inline jekyll/_cci2/security-server.adoc:101]:
```suggestion
* Use Restricted Contexts with teams to share environment variables with a select security group. Read through the https://circleci.com/docs/2.0/contexts/#restricting-a-context[contexts] document to learn more.
```

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**